### DISCUSSION TITLE
[Auth] Support for Steam-TOTP

### DISCUSSION THREAD

**Discussion by borque** [ente-io/ente#1876] [Q&A] [answered]
Hello!
Steam uses a slightly different TOTP. Is it possible to generate these with Ente auth?
If yes: how? If no I would like to propose to add this to a future release.
Thanks and greetings

**Comment by brog-io** [comment, marked as answer]
Ente supports Steam 2FA. You can look at the [guide](https://help.ente.io/auth/migration-guides/steam/) for more info

**Reply by orangepancakes** [reply]
Hello. This method results with regular 6-digit code that doesn't let me log into Steam account. My app version is 3.0.6.

**Reply by brog-io** [reply]
Hey could you update to the latest or 3.0.7? There was an issue in version 2.0.6 and below.

**Reply by orangepancakes** [reply]
> Hey could you update to the latest or 3.0.7? There was an issue in version 2.0.6 and below.

3.0.6 is the latest one that I can get from Appstore. Nevertheless I have found a workaround - instead of scanning QR code as in mentioned guide I have extracted secret key from .maFile created by steamguard-cli and entered it manually into Ente Auth. Now it generates proper codes.
